### PR TITLE
Add Zulip profile

### PR DESCRIPTION
**Pull request by flacks**

### CODE REVIEW

File: etc/zulip.profile [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# Description: Real-time team chat based on the email threading model
+# This file is overwritten after every install/update
+# Persistent local customizations
+include zulip.local
+# Persistent global definitions
+include globals.local
+
+ignore noexec /tmp
+
+noblacklist ${HOME}/.config/Zulip
```
Comment (rusty-snake) [inline etc/zulip.profile]: Add `blacklist ${HOME}/.config/Zulip` to disable-programs.inc (file is alphabetically sorted)

File: etc/zulip.profile [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
```
Comment (rusty-snake) [inline etc/zulip.profile]: What about `include disable-xdg.inc`?

File: etc/zulip.profile [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+nodvd
+nogroups
+nonewprivs
+noroot
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+seccomp
+shell none
```
Comment (rusty-snake) [inline etc/zulip.profile]: What about `tracelog`?

File: etc/zulip.profile [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+disable-mnt
+private-bin locale,zulip
+private-cache
+private-dev
+private-etc asound.conf,fonts,machine-id
```
Comment (rusty-snake) [inline etc/zulip.profile]: Network files like `ca-certificates,ssl,pki,crypto-policies...` not needed?

Comment (flacks) [reply etc/zulip.profile]: Zulip connects to a server just fine without those :man_shrugging:

File: etc/zulip.profile (hunk 3 above)
Comment (flacks) [reply etc/zulip.profile]: Program doesn't seem to run when tracelog is added

File: etc/zulip.profile (hunk 2 above)
Comment (flacks) [reply etc/zulip.profile]: Added with no problems; see corrections commit

File: etc/zulip.profile (hunk 1 above)
Comment (flacks) [reply etc/zulip.profile]: Added; thank you, and sorry, it's been a while since I hacked on firejail :smile:

**Comment by flacks** [comment]
Added zulip to src/firecfg/firecfg.config. See corrections commit.

File: etc/zulip.profile (hunk 4 above)
Comment (rusty-snake) [reply etc/zulip.profile]: then leave it for now

Comment (flacks) [reply etc/zulip.profile]: as is, you mean?

**Comment by rusty-snake** [comment]
Merged, Thanks.

**Pull request merged**